satpy/tests/reader_tests/test_viirs_sdr.py

```diff
@@ -480,19 +488,25 @@ def test_load_all_m_reflectances_provided_geo(self):
             'SVM11_npp_d20120225_t1801245_e1802487_b01708_c20120226002130255476_noaa_ops.h5',
             'GMTCO_npp_d20120225_t1801245_e1802487_b01708_c20120226002130255476_noaa_ops.h5',
         ])
-        r.create_filehandlers(loadables)
-        ds = r.load(['M01',
-                     'M02',
-                     'M03',
-                     'M04',
-                     'M05',
-                     'M06',
-                     'M07',
-                     'M08',
-                     'M09',
-                     'M10',
-                     'M11',
-                     ])
+        geo_fn1 = self._touch_geo_file("GMTCO")
```

Comment:
Should we have a small helper context like:
```python
from contextlib import contextmanager
@contextmanager
def touch_geo_files(*prefixes):
    geofiles = [_touch_geo_file(prefix) for prefix in prefixes]
    try:
        yield
    finally:
        for filename in geofiles:
            os.remove(filename)
```

Reply:
Done.